sections/3-2-work_plan.md

```diff
@@ -1 +1,23 @@
\ No newline at end of file
+Work plan of activities: schedule of pre-meeting activities, agenda of the planned meetings, follow-up plan in order to reach the expected outcomes. As appropriate, please include the source of the data to be synthesized and methods of the analyses. Meetings can be held at any location in Canada; however, preference will be given to meetings hosted at CIEE member organizations. Our experience indicates that a virtual meeting (e.g., via teleconference or Skype) prior to the first physical meeting helps in achieving working group goals. In addition, making real progress typically requires at least four full days of meeting time.
+
+---------
+
+The working group sessions will take place at the University of Regina's Field Station, located in the Center Block of Cypress Hills Inter-Provincial Park. This location was chosen for its centrality within Canada and presence within the ecosystem of interest: the Canadian prairies. 
+
+Meetings will take place over May 8-12th, 2023; pre-meeting materials and logistical information will be circulated to all participants in early March, 2023, and a pre-session meeting with all participants will be scheduled the week of April 3rd-7th. 
+
+**Pre-meeting materials and activities**
+
+In early 2023, the organizers will collect and tidy all database components. This will ensure that working group hours can be spent focused on achieving milestones and generating scientific insight. Over January and February 2023, organizers will select pre-meeting readings to be circulated to all participants in March, and plan/develop the introductory presentation and meeting taking place in early April. In this introductory gathering, to be held over Zoom or an equivalent virtual conferencing platform, organizers will provide a general introduction to themselves and the project, share the daily schedule for the upcoming Working Group meetings, and communicate any actions that participants must take prior to the in-person meeting. A code of conduct will also be developed and shared at this gathering, which will be covered again during the first day of the working group sessions. Participants will be asked to develop a short (~15 minute) introductory presentation to themselves and their work, also to be share on the first day of the upcoming meeting.
+
+**Daily schedule and meeting agendas**
+
+_Monday, May 8th, 2023_
+
+_Tuesday, May 9th, 2023_
+
```

Comment:
Meeting Day 2: Each group will make a quick presentation of their work done the previous day, and then discuss and make a final proposal containing the main hypothesis, model, and preliminary data analysis.